cg/services/order_validation_service/workflows/fastq/validation_service.py

```diff
@@ -46,5 +43,10 @@ def _get_errors(self, raw_order: dict) -> ValidationErrors:
             order_errors=order_errors,
         )
 
-    def parse_and_validate(self, raw_order: dict) -> Order:
-        pass
+    def parse_and_validate(self, raw_order: dict) -> FastqOrder:
+        parsed_order, errors = ModelValidator.validate(order=raw_order, model=FastqOrder)
+        if parsed_order:
+            errors: ValidationErrors = self._perform_rule_validation(order=parsed_order)
+        if not errors.is_empty:
+            raise OrderFormError(message="Order form contained errors")
```

Comment:
I think `OrderFormError` is used when parsing excel sheets and JSONs. Maybe just `OrderError` is ok?

Reply:
Works! I will need to fix the naming clashes since we already have another form of OrderError in this class

Reply:
fixed